portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/fields/FormsSingleSelectionField.testcase

```diff
@@ -113,13 +113,54 @@ definition {
 	}
 
 	@description = "Verify that the Field References for the options can be edited"
-	@ignore = "Test Stub"
 	@priority = "5"
 	test FieldReferencesOfOptionsCanBeEdited {
 		property forms.functional = "true";
 		property portal.acceptance = "false";
 
-		// TODO LPS-132208 FieldReferencesOfOptionsCanBeEdited pending implementation
+		ProductMenu.gotoPortlet(
+			category = "Content &amp; Data",
+			portlet = "Forms");
+
+		FormsAdminNavigator.gotoAddForm();
+
+		Form.editName(formName = "Form 1");
+
+		Form.gotoAddField(
+			fieldPositionNumber = "0",
+			fieldType = "Single Selection");
+
+		Click(locator1 = "Button#SAVE");
+
+		Click(
+			key_fieldName = "Single Selection",
+			locator1 = "Form#FIELD_LABEL");
```

Comment:
There is a macro already created that allows you to edit a Form by its field label.

You can use `FormsAdminNavigator.gotoEditField`, for this scenario it would result in something like `FormsAdminNavigator.gotoEditField(fieldName = "Single Selecion");`

Reply:
Fixed